packages/extension-ui/.eslintrc.js

```diff
@@ -23,7 +23,8 @@ module.exports = {
     '@typescript-eslint/indent': [0],
     'react/jsx-closing-bracket-location': 0,
     'react/jsx-newline': ['warn', { prevent: true, allowMultilines: false }],
-    'sort-keys': 'off'
+    'sort-keys': 'off',
+    'react/jsx-no-bind': 'off',
```

Comment:
This rule is bad. Not memoized callbacks very rarely cause performance problems; what they **do** always cause, on the other hand, is decreased code clarity. The usual rule applies: measure, than optimize.